eng/Dependencies.props

```diff
@@ -94,14 +78,17 @@ and are generated based on the last package release.
     <LatestPackageReference Include="System.Net.Http" Version="$(SystemNetHttpPackageVersion)" />
     <LatestPackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataPackageVersion)" />
     <LatestPackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="$(SystemRuntimeCompilerServicesUnsafePackageVersion)" />
+    <LatestPackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="$(SystemRuntimeInteropServicesRuntimeInformationPackageVersion)" />
```

Comment:
For my understanding - these were packages that (previously) Extensions depended on, but not this repo? And now we're adding a dependency on them since we're bringing in the Extensions stuff that uses them?

Reply:
Yea